File: 프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.java
import java.util.*;

class Solution {
    public int solution(String my_string) {
        String[] tokens = my_string.split(" ");
        int result = Integer.parseInt(tokens[0]);
        for (int i = 1; i < tokens.length; i += 2) {
            int num = Integer.parseInt(tokens[i + 1]);
            if ("+".equals(tokens[i])) result += num;
            else result -= num;
        }
        return result;
    }
}

File: 프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.js
function solution(my_string) {
    const tokens = my_string.split(' ');
    let result = parseInt(tokens[0]);
    for (let i = 1; i < tokens.length; i += 2) {
        const num = parseInt(tokens[i + 1]);
        if (tokens[i] === '+') result += num;
        else result -= num;
    }
    return result;
}
